packages/wallet/backend/src/rapyd/rapyd-client.ts

```diff
@@ -289,7 +289,7 @@ export class RapydClient implements IRapydClient {
       : ','
     const [street, city, postCode] = args.user.address?.split(', ') ?? []
     let address = [street, postCode].join(addressDelimiter)
-    if (args.assetCode === 'EUR') address = address.replace(/[,\s]+/g, '')
+    if (args.assetCode === 'USD') address = address.replace(/[,\s]+/g, '')
```

Comment:
not sure why this transformation happens here tbh, but why was the asset code change needed?

Reply:
Also interested in the answer here, we are creating default USD account for every new user in Rapyd, but don't know if that ihas anything to do with this. But why change EUR?

Reply:
When I first started working on this issue I couldn't create an EUR account in the onboarding and thought that it should be USD by default. This was a change that I introduced in one of the first commits. Since then, I realised that I was wrong about the onboarding but forgot to revert the changes. Now I switched back to EUR 👍